package.json

```diff
@@ -38,7 +38,8 @@
     "react-dom": "18.2.0",
     "react-hook-form": "^7.44.3",
     "react-icons": "^4.9.0",
-    "react-query": "^3.39.3"
+    "react-query": "^3.39.3",
+    "react-router-dom": "^6.14.1"
```

Comment:
Please remove. Next provides routing for us so there's no need to use react-router-dom.